lib/widgets/screens/sign_in/sign_in_screen.dart

```diff
@@ -64,161 +65,342 @@ class _SignInScreenState extends State<SignInScreen> {
+                        },
+                        child: Text(
+                          l10n.createNewAccount,
+                          style: TextStyles.sectionSubHeaderBold(context),
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(Insets.widgetSmallInset),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Container(
+                            width: Dimensions.loginBoxWidth / 2,
+                            height: Dimensions.lineHeight,
+                            color: Colors.black,
+                          ),
+                          Text(
+                            l10n.or,
+                            style: TextStyles.sectionSubNote(context),
+                          ),
+                          Container(
+                            width: Dimensions.loginBoxWidth / 2,
+                            height: Dimensions.lineHeight,
+                            color: Colors.black,
                           ),
                         ],
                       ),
                     ),
-                    const SizedBox(
-                      height: 30,
-                    ),
-                    TextButton(
-                      key: const Key('registerButton'),
-                      onPressed: () {
-                        _openSignUpScreen(context);
-                      },
-                      child: Text(
-                        'Register for a new account',
-                        style: GoogleFonts.poppins(
-                          fontSize: 16,
-                          color: Theme.of(context).colorScheme.onPrimary,
-                          fontWeight: FontWeight.w400,
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: [
+                        Padding(
+                          padding: const EdgeInsets.fromLTRB(
+                              Insets.widgetLargeInset,
+                              Insets.widgetSmallInset,
+                              Insets.widgetSmallInset,
+                              Insets.widgetSmallInset),
+                          child: TextButton(
+                            onPressed: () {
+                              _openSignUpScreen(context);
+                            },
+                            child: Row(
+                              mainAxisSize: MainAxisSize.min,
+                              children: [
+                                const SizedBox(
+                                  width: Dimensions.iconWidth,
+                                  child: Image(
+                                    image:
+                                        AssetImage('assets/images/google.png'),
```

Comment:
Define the asset in the constants library: paths.dart / Assets

Reply:
Awesome, will do!